.github/workflows/posix.yml

```diff
@@ -0,0 +1,93 @@
+name: Posix
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches:
+      - master
+      - 1.2.x
+
+env:
+  PYTEST_WORKERS: "auto"
+  PANDAS_CI: 1
+
+jobs:
+  pytest:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+    strategy:
+      matrix:
+        settings: [
+          [ci/deps/actions-37-minimum_versions.yaml, "not slow and not network and not clipboard", "", "", "", "", ""],
```

Comment:
I think you can define this as before, something like:
```
minumum_versions:
  - ENV_FILE: ci/deps/actions-37-minimum_versions.yaml
  - PATTERN: "not slow and not network and not clipboard"
  ...
```

Do you find this better, or couldn't you implement it this way?

Also, very minor, but instead of repeating `ci/deps/` or `ci/deps/actions-{}.yaml` you could move it below, when you set `ENV_FILE`.

Reply:
> Do you find this better, or couldn't you implement it this way?

Are you sure this can be done with GitHubActions without repeating all the steps? Do you have an example? (I couldn't find one, but I haven't done that much GitHubActions)

> Also, very minor, but instead of repeating ci/deps/ or ci/deps/actions-{}.yaml you could move it below, when you set ENV_FILE.

Good point, thanks!

Reply:
> Are you sure this can be done with GitHubActions without repeating all the steps? Do you have an example? (I couldn't find one, but I haven't done that much GitHubActions)

I think the best way is to use `include`. You've got an example in the docs that should be similar:

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-environment-variables-in-a-matrix